app/test/model/post.test.js

```diff
@@ -11,13 +11,13 @@ test('add, update and delete post', async () => {
     expect(gettedPost).toEqual(post);
 
     post.title = 'tytul2';
-    post.slug = 'slug23';
+    post.slug = '1slug23';
```

Comment:
Nie pisałem ci komentarza pod każdym takim wystąpieniem. Przywróć to też

Reply:
poprawione